sqlmesh/core/config/connection.py

```diff
@@ -201,6 +202,7 @@ class BaseDuckDBConnectionConfig(ConnectionConfig):
         catalogs: Key is the name of the catalog and value is the path.
         extensions: A list of autoloadable extensions to load.
         connector_config: A dictionary of configuration to pass into the duckdb connector.
+        secrets_config: A list of dictionaries used to generate DuckDB secrets for authenticating with external services (e.g. S3).
```

Comment:
`_config` seems redundant since this is defined in the config already. Can this be just `secrets`?